Doc/library/stdtypes.rst

```diff
@@ -4237,6 +4237,11 @@ pairs within braces, for example: ``{'jack': 4098, 'sjoerd': 4127}`` or ``{4098:
          LIFO order is now guaranteed. In prior versions, :meth:`popitem` would
          return an arbitrary key/value pair.
 
+   .. describe:: reversed(d)
+
+      Return an reversed iterator over the keys of the dictionnary. This is a
+      shortcut for ``reversed(d.keys())``.
+
```

Comment:
This section should be right after `describe:: iter(d)`.
``versionadded::`` is also helpful.

Reply:
Looking at other sections of the documentation, I thought it was supposed to be ordered alphabetically, isn't that so?